python-sdk/tests/test_example_dags.py

```diff
@@ -47,48 +51,43 @@ def session():
     return get_session()
 
 
-@pytest.mark.parametrize(
-    "dag_id",
-    [
-        "example_amazon_s3_postgres",
-        "example_amazon_s3_postgres_load_and_save",
-        "example_amazon_s3_snowflake_transform",
-        "example_google_bigquery_gcs_load_and_save",
-        "example_snowflake_partial_table_with_append",
-        "example_sqlite_load_transform",
-        "example_append",
-        "example_load_file",
-        "example_transform",
-        "example_merge_bigquery",
-        "example_transform_file",
-        "calculate_popular_movies",
-    ],
-)
-def test_example_dag(session, dag_id):
-    dir_path = os.path.dirname(os.path.realpath(__file__))
-    db = DagBag(dir_path + "/../example_dags")
-    dag = db.get_dag(dag_id)
+DAG_BAG = DagBag(Path(__file__).parent.parent / "example_dags", include_examples=False)
```

Comment:
This looks good, last thought, how can we protect against dag parsing errors in one of the dags?

For example, if `example_amazon_s3_postgres` dag has a parsing error, currently it will be skipped from running the test.

Reply:
Added a test for import errors 8f0f067